docs/user/AzureHelpTopics.md

```diff
@@ -24,7 +25,23 @@ az login
 ]
 ```
 
-You can also use the ```az account list``` command to list all the subscriptions and tenants you belong to. From Azure CLI commands, the value of `id` is used in `TF_VAR_subscription_id` and `tenantId` is used in `TF_VAR_tenant_id`.
+## How to establish the TF environment variables:
+
+You can use the following commands, or alter them as needed to establish the environment variables with your `Tenant ID` and `Subscription`:
+
+```bash
+# the following commands should work and should establish `TF_VAR_subscription_id` and `TF_VAR_tenant_id`
```

Comment:
Line 33-34 feel like they should be outside the code block as the are not code. There are descriptors for said code block

Reply:
In the event that a user took the bash commands and created their own script, I included the comments inline so that the comments and the code went together as a package.  If you need, I can move those above the code block.

Reply:
makes sense